silkworm/silkrpc/json/types.cpp

```diff
@@ -211,6 +211,11 @@ void to_json(nlohmann::json& json, const BlockHeader& header) {
     json["gasLimit"] = rpc::to_quantity(header.gas_limit);
     json["gasUsed"] = rpc::to_quantity(header.gas_used);
     json["timestamp"] = rpc::to_quantity(header.timestamp);
+    if (header.consensus_parameter_index.has_value()) {
```

Comment:
Maybe we should return the json object with all the values instead of just the index?